### ISSUE TITLE
Accessibility / ARIA implementation : multiple issues

### ISSUE THREAD

**Issue by wayofspark** [qTip2/qTip2#729]
Hello

Some issues regarding your ARIA implementation on qtip2 : 

1/ role should not be "alert", but "tooltip", as specified in the ARIA specification, 
here : http://www.w3.org/WAI/PF/aria/roles#tooltip 
and here : http://www.w3.org/WAI/PF/aria-practices/ ( chapter 3.2.6.1 )

2/ In the same time, the aria-live attribute should be removed, as it is too obstrusive, even in "polite" mode. For example, in actual implementation, tooltips activating on focus make the screen reader read twice the tooltip content : once because of aria-live, and second time because the focused element is aria-describedby the tooltip content. (also, aria-live on tooltips is not even suggested by the specification)
The aria-describedby on tooltip's target is the only thing that matter here.

3/ aria-describedby can support MULTIPLE ids ( aka aria-describedby="id1 id2 id3" ). 
see http://www.w3.org/WAI/PF/aria/states_and_properties#aria-describedby
In your current implementation, adding a tooltip to an element already having an aria-describedby attribute erase the current id references instead of adding the tooltip's id to it (same behaviour when the tooltip is destroyed : the original aria-describedby value is lost)

**Comment by stalsma** [comment]
I second this.  While #3 is not so much an issue for me, my poor users are getting bombarded w/definitions reading 2-3 times.
